deployments/docker/state-offline-installer/centos/Dockerfile

```diff
@@ -0,0 +1,41 @@
+ARG CENTOS_VERSION=centos-7.6-builder:1.0.67
```

Comment:
What is this Docker accomplishing exactly? I feel like this should exist entirely within the activestate.yaml (see the build scripts we have).

Relying on Docker feels like working around using the State Tool in this case.